src/RoadRegistry.BackOffice/Messages/RoadNetworkExtractGotRequestedV2.cs

```diff
@@ -5,12 +5,21 @@ namespace RoadRegistry.BackOffice.Messages;
 
 [EventName("RoadNetworkExtractGotRequestedV2")]
 [EventDescription("Indicates a road network extract was requested.")]
-public class RoadNetworkExtractGotRequestedV2 : IMessage
+public class RoadNetworkExtractGotRequestedV2 : IRoadNetworkExtractGotRequestedMessage
 {
     public RoadNetworkExtractGeometry Contour { get; set; }
     public string Description { get; set; }
     public Guid DownloadId { get; set; }
     public string ExternalRequestId { get; set; }
     public string RequestId { get; set; }
     public string When { get; set; }
-}
\ No newline at end of file
+}
+
+public interface IRoadNetworkExtractGotRequestedMessage : IMessage
+{
+    public RoadNetworkExtractGeometry Contour { get; set; }
```

Comment:
`public` accessor is not relevant for interfaces

Reply:
Moved into own file in Core and removed accessor